service/sensor.go

```diff
@@ -12,13 +12,20 @@ import (
 	"gopkg.in/mgo.v2/bson"
 )
 
-var sensorRepository = repository.SensorRepository{}
-
-func init() {
-	sensorRepository.Connect()
+type ISersoService interface {
+	Create(w http.ResponseWriter, r *http.Request)
+	FindAll(w http.ResponseWriter, r *http.Request)
+	FindOne(w http.ResponseWriter, r *http.Request)
+	Delete(w http.ResponseWriter, r *http.Request)
+	Update(w http.ResponseWriter, r *http.Request)
+	VerifyIntegrityUrlSensor(id string) bool
+	GeneratingUrlSensors() ([]string, error)
```

Comment:
@adrissonGalvao please rename  "GeneratingUrlSensors", this name don't fit in your names pattern.